data/obs-packaging/README.md

```diff
@@ -0,0 +1,4 @@
+#Clear Containers OBS packages
+
+
+WIP
```

Comment:
This top-level README really needs some text to explain what this is all about.

I'd add a brief summary explaining what OBS is (including a link to the main site) and why we use it.